plugins/polio/js/src/pages/LQAS/LqasAfroOverview/Map/LqasAfroMapPanesContainer.tsx

```diff
@@ -20,7 +28,7 @@ const getBackgroundLayerStyle = () => {
         color: '#5e5e5e',
         opacity: '1',
         fillColor: 'lightGrey',
-        weight: '2',
+        weight: '4',
```

Comment:
4 is a bit much when viewing districts while zoomed out
![Screenshot 2023-08-22 at 16 18 50](https://github.com/BLSQ/iaso/assets/38907762/9e4586a4-e6ee-4f74-8197-2560ee643f68)